www-client/firefox/firefox-137.0.2.ebuild

```diff
@@ -600,6 +600,7 @@ src_prepare() {
 
 	eapply "${WORKDIR}/firefox-patches"
 	use loong && eapply "${WORKDIR}/firefox-loong-patches"
+	use jpegxl && eapply "${FILESDIR}/firefox-133.0.3-jxl-enable.patch"
```

Comment:
No, the patch needs to be always applied and `--disable-jxl` used instead based on the use flag.